packages/esbuild/lib/index.js

```diff
@@ -13,7 +13,9 @@
  * @typedef {ProcessorOptions & {allowDangerousRemoteMdx?: boolean}} Options
  */
 
+import assert from 'node:assert'
 import {promises as fs} from 'fs'
```

Comment:
Would it make sense for `fs` to also have a `node:` prefix?

Reply:
Good point, I’ll remove the `node:` prefixes.
There was some reason why I didn’t apply them here (they are in xdm). Perhaps due to CRA or Next or so?

Reply:
hmm, it’s a bit inconsistent now anyway. I’ve made it more consistent. esbuild and the node loader probably run in a Node version that support it. I’ve “downgraded” one instance of MDX core though.